backend/src/core/services/auth.service.ts

```diff
@@ -189,15 +193,16 @@ const sendOtp = async (email: string): Promise<string | void> => {
   }
   await saveHashedOtp(email, hashedOtp)
 
+  const appName = config.get('APP_NAME')
   return MailService.mailClient.sendMail({
     recipients: [email],
     subject: `One-Time Password (OTP) for ${config.get('APP_NAME')}`,
     body: `Your OTP is <b>${otp}</b>. It will expire in ${Math.floor(
       OTP_EXPIRY / 60
     )} minutes.
-    Please use this to login to your ${config.get(
-      'APP_NAME'
-    )} account. <p>If your OTP does not work, please request for a new OTP.</p>`,
+    Please use this to login to your ${appName} account. <p>If your OTP does not work, please request for a new OTP.</p>
+    <p>This login attempt was made from the IP: ${ipAddress}. If you did not attempt to log in to ${appName}, you may choose to investigate this IP address further.</p>
```

Comment:
This is meaningless right? What's the method of investigation?

Reply:
Good point. Should we change it to get them to reach out to us? We can then validate and potentially block the IP if it's a legit threat.